.github/workflows/molecule_tests.yml

```diff
@@ -0,0 +1,37 @@
+---
+# SPDX-license-identifier: Apache-2.0
+##############################################################################
+# Copyright (c) 2021
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+
+name: Check Molecule Tests
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+
+jobs:
+  check-molecule:
+    runs-on: macos-10.15
+    strategy:
+      fail-fast: false
+      matrix:
+        tox_env: [ovs_prepare]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache Vagrant boxes
```

Comment:
Please drop the use of `actions/cache@v2`.  It was smart of you to cache, but unfortunately our project has a limit of 5GB of cache for GH Workflows and Vagrant / Docker builds will easily swallow that up and then some.  We are reserving the limited cache space for e.g. build tooling such as C++ caching (smaller needs, high impact on average PR).